e2e/tests/components/number-picker/number-picker.e2e-spec.ts

```diff
@@ -156,4 +156,28 @@ describe('Number Picker Tests', () => {
         expect(await page.confirmUpDownControlIsDisabled(page.numberPicker2, 'down')).toBeFalsy();
 
     });
+
+    it('should update validation when value changes to max or min', async () => {
+
+        // Number Picker 1
+        await page.setNumberPickerValue(page.numberPicker1, '11');
+        expect(page.numberPicker1.getAttribute('class')).toContain('ux-number-picker-invalid');
+        await page.decrementNumberPickerValue(page.numberPicker1);
+        expect(page.numberPicker1.getAttribute('class')).not.toContain('ux-number-picker-invalid');
+        await page.setNumberPickerValue(page.numberPicker1, '-11');
+        expect(page.numberPicker1.getAttribute('class')).toContain('ux-number-picker-invalid');
+        await page.incrementNumberPickerValue(page.numberPicker1);
+        expect(page.numberPicker1.getAttribute('class')).not.toContain('ux-number-picker-invalid');
+
+        // Number Picker 2
+        await page.setNumberPickerValue(page.numberPicker2, '10.1');
+        expect(page.numberPicker2.getAttribute('class')).toContain('ux-number-picker-invalid');
+        await page.decrementNumberPickerValue(page.numberPicker2);
+        expect(page.numberPicker2.getAttribute('class')).not.toContain('ux-number-picker-invalid');
+        await page.setNumberPickerValue(page.numberPicker2, '-10.1');
+        expect(page.numberPicker2.getAttribute('class')).toContain('ux-number-picker-invalid');
+        await page.incrementNumberPickerValue(page.numberPicker2);
+        expect(page.numberPicker2.getAttribute('class')).not.toContain('ux-number-picker-invalid');

```

Comment:
Possibly a misunderstanding of the request, since this test doesn't modify the max or min inputs. Try implementing "changes to min and max should update the validation state" as a test instead.